spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'ingredient index', type: :feature do
+  before(:each) do
+    @ingredient_1 = Ingredient.create!(name: "Beef", cost: 40)
+    @ingredient_2 = Ingredient.create!(name: "Chicken", cost: 20)
+  end
+
+  # User Story 1 - Ingredients Index
+  it "ingredients attributes" do
+    # As a visitor,
+    # When I visit '/ingredients'
+    visit "/ingredients"
+    # I see a list of all the ingredients including their name and cost
+    expect(page).to have_content(@ingredient_1.name)
```

Comment:
Nice job with your test setup, and breaking down the user story line-by-line. I think doing your tests this way helps to make sure you're not missing any details! Keep this up 👍

Reply:
You may want to try a `within` block around these statements as well, for practice ensuring the content appears precisely where you want it to.